pyproject.toml

```diff
@@ -108,7 +108,7 @@ SQLAlchemy = ">=1.3.3,<1.5.0"
 sklearn-crfsuite = "~0.3"
 psycopg2-binary = ">=2.8.2,<2.10.0"
 python-dateutil = "~2.8"
-tensorflow = ">=2.6,<2.7"
+tensorflow = "~2.6.2"
```

Comment:
The `tensorflow` and `google-auth` dependencies have been set to the respective versions b/c of a micro release after a customer reported problems.

This is just to double check whether we want to merge these back into `main` (b/c idk)?

cc @hsm207 @tayfun

Reply:
In my opinion, this should be merge to main because if we don't do it, customers will report the same tensorflow vulnerabilities when they upgrade to 3.x.